media-plugins/audacious-plugins/audacious-plugins-4.4.ebuild

```diff
@@ -0,0 +1,199 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="${P/_/-}"
+
+inherit meson
+
+DESCRIPTION="Lightweight and versatile audio player"
+HOMEPAGE="https://audacious-media-player.org/"
+SRC_URI="https://distfiles.audacious-media-player.org/${MY_P}.tar.bz2"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+IUSE="
+	aac +alsa ampache bs2b cdda cue ffmpeg flac fluidsynth gme http jack
+	lame libnotify libsamplerate lirc mms modplug mp3 opengl openmpt
+	opus pipewire pulseaudio scrobbler sdl sid sndfile soxr speedpitch
+	streamtuner vorbis wavpack X
+"
+
+REQUIRED_USE="
+	ampache? ( http )
+	streamtuner? ( http )
+"
+
+# The following plugins REQUIRE a GUI build of audacious, because non-GUI
+# builds do NOT install the libaudgui library & headers.
+# Plugins without a configure option:
+#   alarm
+#   albumart
+#   delete-files
+#   ladspa
+#   playlist-manager
+#   search-tool
+#   skins
+#   vtx
+# Plugins with a configure option:
+#   glspectrum
+#   gtkui
+#   hotkey
+#   notify
+#   statusicon
+BDEPEND="
+	dev-util/gdbus-codegen
+	virtual/pkgconfig
+"
+# Add this dep on the next bump when Qt6 support is available for -Dqtaudio
+#	dev-qt/qtmultimedia:6=
+DEPEND="
+	app-arch/unzip
+	dev-libs/glib:2
+	>=dev-libs/json-glib-1.0
+	dev-libs/libxml2:2
+	dev-qt/qtbase:6=[gui,opengl?,widgets]
```

Comment:
same here re private includes and on any other `dev-qt/*` dependency (see subslot metadata).

Reply:
Ah, thanks. I suspect not, but I will check.

Reply:
(most prevalent header to look out for is `qtx11extras_p.h`, actually, requiring `dev-qt/qtbase[X]`)

Reply:
Yeah, I see no evidence of using private Qt headers, so I've dropped the `=`.